python/paddle/trainer_config_helpers/layers.py

```diff
@@ -5593,7 +5595,7 @@ def rank_cost(left,
     :param weight: The weight affects the cost, namely the scale of cost.
                    It is an optional argument.
     :type weight: LayerOutput
-    :param name: The name of this layers. It is not necessary.
+    :param name: The name of this layer. It is not necessary.
```

Comment:
please change the sentence "It is not necessary." into "It is not required."
**The same below.**